.github/workflows/simorgh-misc-checks.yml

```diff
@@ -0,0 +1,40 @@
+name: Simorgh CI - Miscellaneous Checks
```

Comment:
Mind it we give this more descriptive name e.g. Licences, Dependencies, Lint, Chromatic UI (I know it's a mouthful - it's just easier to see on the PR)

Reply:
Updated